src/redis-pubsub.ts

```diff
@@ -1,5 +1,6 @@
+import * as IORedis from 'ioredis';
+import { RedisOptions, Redis as RedisClient } from 'ioredis';
```

Comment:
Due to the fact that it's [deprecated](https://github.com/luin/ioredis/blob/f6fdc8234dceed2c0b34fe562a4e7021b476547c/lib/redis.js#L138) Could you please use the recommended way of creating a client with ioredis?